index.html

```diff
@@ -33,52 +34,81 @@
   <script src="https://kit.fontawesome.com/d14f08e7fd.js" crossorigin="anonymous"></script>
 
 
-
-
-
   <link rel="stylesheet" href="assets/css/style.css">
   <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bulma@0.9.2/css/bulma.min.css">
+  <link rel="preconnect" href="https://fonts.gstatic.com">
```

Comment:
Which fonts are we using from fonts.gstatic.com? I think we don't need this either. Please remove this as well.

Reply:
done